src/main/features/response/draft/responseDraft.ts

```diff
@@ -13,7 +13,6 @@ import { QualifiedStatementOfTruth } from 'app/forms/models/qualifiedStatementOf
 import { HowMuchPaid } from 'response/form/models/howMuchPaid'
 import { HowMuchOwed } from 'response/form/models/howMuchOwed'
 import { Timeline } from 'response/form/models/timeline'
-import { Evidence } from 'response/form/models/evidence'
```

Comment:
Why did you remove it?

Reply:
That was a mistake. I meant to add that back. Ill do that now.

Reply:
On a merge with origin it said it wasn't used so i removed it. Its been added back now.